### PR TITLE
fix most of the typos

### PR DESCRIPTION
**Pull request by lhfbc**
## Motivation

fix typos

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

no new feature and 'cargo test' passed

## Related PRs

none

### CODE REVIEW

**Comment by Nikolaj-K** [comment]
👍 

---

As a semi-related remark, I think line breaks before and after links in the readme markdowns help readability and could be added. I.e. writing 

```
the text like this with
[some description with a link to some page](https://github.com/libra/libra/pull/219)
like this
```

instead of

```
the text like this with [some description with a link to some page](https://github.com/libra/libra/pull/219) like this
```

As displayed on github, those will appear exactly the same but the later kind of markdown is harder to read, especially if the files link to 5 different web pages in one paragraph. 
E.g. kind of text wall is not necessary:

image

**Comment by lhfbc** [comment]
I rebased this branch from master to fix the conflict and modified the readme as suggested.

**Comment by lhfbc** [comment]
> Resolve confit

I rebased it again.

File: consensus/src/chained_bft/safety/safety_rules_test.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ cached_key! {
     // map's state.
     LENGTH: SizedCache<String, (usize, Vec<HashValue>)> = SizedCache::with_size(50);
     Key = { format!("{}{:?}{:?}", calculate_hash(children_table), query, initial_contiguous_links ) };
-    // This returns the length of the maximal chain constructible from the
+    // This returns the length of the maximal chain constructable from the
```
Comment (huitseeker) [inline consensus/src/chained_bft/safety/safety_rules_test.rs]: This doesn't look [right](https://en.wikipedia.org/wiki/Constructibility).

File: network/netcore/src/negotiate/outbound.rs [hunk 2]
```diff
@@ -25,7 +25,7 @@ where
     write_u16frame(&mut stream, PROTOCOL_INTERACTIVE).await?;
 
     let mut buf = BytesMut::new();
-    let mut recieved_header_ack = false;
+    let mut received_header_ace = false;
```
Comment (huitseeker) [inline network/netcore/src/negotiate/outbound.rs]: The three occurrences of `recieved_header_ack` are fine here.

**Comment by huitseeker** [comment]
@lhfbc 
Thanks for the contribution! This looks like a good change, but needs a rebase.

However, due to some CI changes put in place this past week, to get CI to run we would need you to push against your branch again (e.g  in a rebase), or to close this PR and make a new one. Going forward this won't be an issue, but unfortunately, we do not have a nice way to force CI to run for older PRs.

**Comment by davidiw** [comment]
Please re-base and re-request if still valid.

**Pull request closed without merging**